README.md

```diff
@@ -1,10 +1,14 @@
 <p align="center">
-  <a href="http://chanjo.co">
+  <a href="https://github.com/Clinical-Genomics/chanjo">
     <img height="235" width="244" src="docs/assets/logo.png"/>
   </a>
 </p>
 
-# Chanjo [![PyPI version][fury-image]][fury-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url] [![bioconda-badge][bioconda-img]][bioconda-url]
+# Chanjo 
+![Docker build - GitHub](https://github.com/Clinical-Genomics/chanjo/actions/workflows/docker_build_n_publish.yml/badge.svg)
```

Comment:
I don't see an image for this, but maybe it has just been purged since no updates, and will manifest on next run?

Reply:
That's because the action never had a run: https://github.com/Clinical-Genomics/chanjo/actions/workflows/docker_build_n_publish.yml